modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItDmlTest.java

```diff
@@ -350,6 +351,33 @@ public void testMergeKeysConflict() {
         assertEquals(Sql.DUPLICATE_KEYS_ERR, ex.code());
     }
 
+    /**
+     * Test verifies that scan is executed within provided transaction.
+     */
+    @Test
+    @Disabled("https://issues.apache.org/jira/browse/IGNITE-17309")
```

Comment:
the ticket is resolved, should be there another one?

Reply:
This ticked is merged to a feature branch, but not main. Well, probably it does make sense to change this link to a link to epic...